File: ProcessoConcrete/app/src/main/java/br/com/guilherme/concrete/model/Repositorio.java
package br.com.guilherme.concrete.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Repositorio{
    @SerializedName("owner")
    private Usuario user;

    @SerializedName("items")
    private List<Repositorio> repositorios;

    @SerializedName("name")
    private String nomeRepositorio;

    @SerializedName("description")
    private String descricaoRespositorio;

    @SerializedName("forks")
    private int qtdBranches;

    @SerializedName("stargazers_count")
    private int qtdFavoritos;

    // Getters and Setters
    public String getNomeRepositorio() {
        return nomeRepositorio;
    }

    public String getDescricaoRespositorio() {
        return descricaoRespositorio;
    }

    public int getQtdBranches() {
        return qtdBranches;
    }

    public int getQtdFavoritos() {
        return qtdFavoritos;
    }

    public List<Repositorio> getRepositorios() {
        return repositorios;
    }

    public Usuario getUser() {
        return user;
    }
}

File: app/src/main/java/br/com/eriberto/desafioandroidconcrete/model/pojo/Repositorio.kt
package br.com.eriberto.desafioandroidconcrete.model.pojo

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Repositorio(
    @SerializedName("name")
    var nomeRepositorio: String,
    @SerializedName("description")
    var descricaoRepositorio: String,
    @SerializedName("owner")
    var proprietario: Proprietario,
    @SerializedName("forks_count")
    var quantidadeDeEstrelas: Long,
    @SerializedName("forks")
    var quantidadeDeForks: Long,
    @SerializedName("html_url")
    var htmlSite: String
) : Serializable